package/MDAnalysis/analysis/density.py

```diff
@@ -511,7 +512,13 @@ def current_coordinates():
     for ts in u.trajectory:
         print("Histograming %6d atoms in frame %5d/%d  [%5.1f%%]\r" % \
```

Comment:
The `print` should be replaced with use of `lib.log.ProgressMeter` — @Balasubra you don't have to do this unless you want to but it should eventually be changed for consistency.

```python
from MDAnalysis.lib.log import ProgressMeter

# ....

pm = ProgressMeter(u.trajectory.n_frames, 
       format="Histograming %(n_atoms)6d atoms in frame %(step)5d/%(numsteps)d  [%(percentage)5.1f%%]\r")
for ts in u.trajectory:
    # ...
    # ...
    if len(coord) == 0:
             continue
    pm.echo(ts.frame, n_atoms=group.n_atoms)
    h[:], edges[:] = np.histogramdd(coord, bins=bins, range=arange, normed=False)
```